config/crd/bases/opentelemetry.io_opentelemetrycollectors.yaml

```diff
@@ -249,10 +247,8 @@ spec:
                   attached to Collector and Target Allocator pods.
                 type: object
               podSecurityContext:
-                description: PodSecurityContext holds pod-level security attributes
-                  and common container settings. Some fields are also present in container.securityContext.  Field
-                  values of container.securityContext take precedence over field values
-                  of PodSecurityContext.
+                description: SecurityContext will be set as the container security
```

Comment:
Which version did you use to create the bundle?

Reply:
Oh sorry! I found the problem (I though the change was expected but not). I used the `fix` option from `fieldalignment` and it removes the descriptions. I fixed some of them manually but it seems I broke others. Working on the fix.

Thanks for the feedback and sorry for the inconvenience :)